blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/OrbManagement.java

```diff
@@ -26,9 +26,9 @@
 public class OrbManagement implements Runnable {
     private static final Logger log = LogManager.getLogger(OrbManagement.class);
     private static final String CorbaOrbClassProp = "org.omg.CORBA.ORBClass";
-    private static final String CorbaOrbClassValue = "org.jacorb.orb.ORB";
+    private static final String CorbaOrbClassValue = "com.sun.corba.se.impl.orb.ORBImpl";
```

Comment:
Sure, but I think we should really try to consider whether we need OrbManagement at all as the code should not really be using ORB anymore as we use RTS to propagate the transaction. There is one use case within the app server were the transaction is propagated over EJB and if we need OrbManagement for that then sure, if the change you made works then happy to go with that. If we don't need the OrbManagement class for that I would just remove it but that is my personal opinion and I haven't reviewed the code itself.